docs/published/prize_motivation.tex

```diff
@@ -69,15 +69,14 @@ \subsection{Kindle Technologies - Fit for Purpose}
 authentication.
 
 \subsection{Quant - Most innovative use of open source}
-\docks utilises only open soure software. Docker is completely open source and all
-of the technologies and frameworks we used are also open source. 
-
-%Help met die asseblief
-A member our out group, Evert Geldenhuys, also wrote a package that is freely 
-available to the public community. \docker has the functionality where a compose file
-can be used to generate a stack and services but once the stack has been deployed the 
-information in the compose file cannot be extracted again. Evert wrote a package that
-allows you to generate the compose file from a running stack in the swarm.
+% We are still 5 words over budget.. not sure where to trim
+\docks is a completely open source project to enhance an existing open source tool and is licensed under the copyleft \textit{GNU General Public License v3.0}. Our project extends the existing \href{https://github.com/docker}{open source Docker platform} by adding advanced functionality to the \href{https://github.com/docker/swarm}{Docker Swarm} container orchestration project. \\
```

Comment:
This will clarify that we did not contribute to their code base, but instead integrated with it.

```suggestion
\docks is a completely open source project to enhance an existing open source tool and is licensed under the copyleft \textit{GNU General Public License v3.0}. Our project extends the existing \href{https://github.com/docker}{open source Docker platform} by integrating with the \href{https://github.com/docker/swarm}{Docker Swarm} container orchestration project's API to provide advanced functionality.\\
```